builder/vmware/common/driver.go

```diff
@@ -642,8 +647,46 @@ func GetOVFTool() string {
 	return ovftool
 }
 
+func CheckOvfToolVersion(ovftoolPath string) bool {
```

Comment:
That'd be slightly more complex, but since there are multiple occasions where the function fails, I would suggest returning `(bool, error)` so that there's a difference between "installed at wrong version", and "failed to get the version".

That or just `error` even, with one error being `incompatible version of OVFTool`, I'll let you pick which suits you best

Reply:
Resolved in 939a252.